go.mod

```diff
@@ -202,7 +202,7 @@ require (
 	gopkg.in/evanphx/json-patch.v4 v4.12.0 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/apiextensions-apiserver v0.32.1 // indirect
+	k8s.io/apiextensions-apiserver v0.32.1
```

Comment:
no changes in go.sum?

Reply:
No changes. There was indirect dependence on `apiextensions-apiserver` library before, but now it is directly imported, so this `// indirect` directive has been removed as a result of `go mod tidy` operation.